hasjob/views/index.py

```diff
@@ -861,3 +864,57 @@ def manifest():
 @app.route('/api/1/embed.js', methods=['GET'])
 def embed():
     return app.send_static_file('embed.js')
+
+
+@app.route('/api/1/oembed', methods=['GET'], subdomain='<subdomain>')
+@app.route('/api/1/oembed', methods=['GET'])
+@requestargs('url')
+def oembed(url):
+    """
+    Endpoint to support oEmbed - https://oembed.com/
+
+    Example request - http://hasjob.co/api/1/oembed?url=https://hasjob.co/
+
+    Required for services like embed.ly
+    """
+    oembedjs = {}
+
+    parsed = urlparse(url)
+    if not parsed.netloc.endswith(app.config['SERVER_NAME']):
+        # is this a legit hasjob domain or subdomain? if not, then return 404
+        abort(404)
+
+    boardname, _ = parsed.netloc.split('.', 1)
+    if boardname == 'hasjob':
+        # in case the url is https://hasjob.co/
+        boardname = 'www'
+
+    board = Board.get(boardname)
+    if not board:
+        abort(404)
+
+    try:
+        adapter = app.url_map.bind_to_environ(request)
+        url_name, _ = adapter.match(parsed.path)
```

Comment:
`_` is reserved for i18n. Don't use it for temporary variables. Also, it returns the arguments here and will include the subdomain, which you want.